data.sql

```diff
@@ -31,4 +31,55 @@ UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer
 UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob') where name = 'Devimon' or name='Plantmon';
 
 UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') where name = 'Charmander' or name ='Squirtle' or name='Blossom';
-UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') where name = 'Angemon' or name='Boarmon';
\ No newline at end of file
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') where name = 'Angemon' or name='Boarmon';
+
+
+INSERT INTO vets(id, name, age, date_of_graduation) VALUES (1, 'William Tatcher', 45, '2000-04-23');
+INSERT INTO vets(id, name, age, date_of_graduation) VALUES (2, 'Maisy Smith', 26, '2019-01-17');
+INSERT INTO vets(id, name, age, date_of_graduation) VALUES (3, 'Stephanie', 64, '1981-05-04');
```

Comment:
- [ ] Great job so far 👍 Kindly add Jack Harknes as he is the fourth vet into the vets table 👍